python/cudf/tests/test_parquet.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+import cudf
+from cudf.tests.utils import assert_eq
+
+import pytest
+import os
+import pandas as pd
+
+
+@pytest.fixture
+def pdf():
+    return pd.DataFrame({'x': range(10),
+                         'y': range(10)})
+
+
+@pytest.fixture
+def gdf(pdf):
+    return cudf.DataFrame.from_pandas(pdf)
+
+
+@pytest.fixture(params=['snappy', 'gzip', 'botli', None])
+def parquet_file(request, tmp_path_factory, pdf):
+    fname = tmp_path_factory.mktemp("parquet") / "test.parquet"
+    pdf.to_parquet(fname, engine='pyarrow', compression=request.param)
```

Comment:
This is `snappy` every time because of above.

Reply:
I'm not following here, the pytests show the compression param is being picked up properly based on the test names.